setup.py

```diff
@@ -63,7 +73,6 @@ def remove_prefix(name, prefix):
 logging.basicConfig(level=getattr(logging,args[0].log),format='%(levelname)s:%(message)s',filename=args[0].log_file)
 logging.debug('setup.py called with the following optional args\n %s\n argument parsing completed.',vars(args[0]))
 try:
-    from subprocess import Popen, PIPE
     _p = Popen(["svnversion", "."], stdout=PIPE)
     revision = _p.communicate()[0].decode('ascii')
```

Comment:
If you don't mind you could remove this entire try/except, it is an old artifact from before assimulo moved to git.

Reply:
ok